align_system/algorithms/misc_itm_adm_components.py

```diff
@@ -162,3 +163,55 @@ def run_returns(self):
 
     def run(self):
         return "Looked at scores."
+
+
+class Phase2RegressionRemoveIrrelevantAttributes(ADMComponent):
+    def run_returns(self):
+        return ('attribute_prediction_scores',
+                'alignment_target')
+
+    def run(self,
+            attribute_prediction_scores,
+            alignment_target
+            ):
+        # If there are two non-medical attributes, removes the one with smaller delta
+
+        attributes = list(next(iter(attribute_prediction_scores.values())).keys())
```

Comment:
This probably works fine most of the time, the safest thing would be to build up a `set` of all keys for the values so we don't miss any.  (That said maybe it's an error if we don't have the same attributes for all choices)